src/Api/API.php

```diff
@@ -2522,7 +2519,7 @@ protected static function parseDropdowns($fields, $params = [])
 
                 if (
                     !empty($value)
-                    || $key == 'entities_id' && $value >= 0
+                    || $key == 'entities_id' && !is_array($value) && $value >= 0
```

Comment:
I do not understand what it would change; and a sit's not required for phpstan checks to turn green, I prefer not to change anything in API

Reply:
I think PHPStan is warning about a potentiel array value because `parseDropdowns()` always returns an array, so `$value = self::parseDropdowns($value, $params);` will always be an array.
In this case, the whole code present in the `if (isForeignKeyField($key)) {` will have no effect because the latest `if` block conditions will be false. My proposal was just to add a `continue` ealier to make the code more clear, but it would have the same effect as tur proposal.